controllers/drplacementcontrol_controller_test.go

```diff
@@ -746,5 +746,34 @@ var _ = Describe("DRPlacementControl Reconciler", func() {
 				waitForCompletion()
 			})
 		})
+		When("DRAction is set to failback", func() {
```

Comment:
(nit) "failback" -> "relocate" in test case description.

Reply:
good catch